packages/prettier-config/index.json

```diff
@@ -1,15 +1,5 @@
 {
   "$schema": "https://json.schemastore.org/prettierrc.json",
-  "arrowParens": "avoid",
-  "endOfLine": "auto",
-  "printWidth": 120,
-  "quoteProps": "preserve",
-  "tabWidth": 4,
-  "trailingComma": "none",
-  "overrides": [
-    {
-      "files": ["*.json", "*.jsonc", "*.json5", "*.yml"],
-      "options": { "tabWidth": 2 }
-    }
-  ]
```

Comment:
Since the default uses `.editorconfig` but `yarn` and other tooling always does 2 spaces, maybe we should leave this or just make it `tabWidth: 2`.

I'm OK with leaving it, just wanted to mention my memory of why this is here.

Reply:
@101100 

For me, `.editorconfig` is responsible for configuring tabbing of all files that are present in the project. `prettier` must not override this configuration, because this will lead to inconsistencies where editor would use one indentation setting while making changes to a file, just to be replaced with another indentation when `prettier` reformats the file during save operation.

This means that if a project wants to use 4-space indentation in code files but also has 2-space files like YML, the project's `.editorconfig` must have extra sections to override the default 4-space indentation as needed.